core/src/test/scala/org/apache/spark/sql/delta/MergeIntoSuiteBase.scala

```diff
@@ -676,23 +676,6 @@ abstract class MergeIntoSuiteBase
     }
   }
 
-  test("Negative case - non-delta target") {
-    withTable("source", "target") {
-      Seq((1, 1), (0, 3), (1, 5)).toDF("key1", "value").createOrReplaceTempView("source")
-      Seq((1, 1), (0, 3), (1, 5)).toDF("key2", "value").write.saveAsTable("target")
-
-      val e = intercept[AnalysisException] {
-        executeMerge(
-          target = "target",
-          source = "source src",
-          condition = "src.key1 = target.key2",
-          update = "key2 = 20 + key1, value = 20 + src.value",
-          insert = "(key2, value) VALUES (key1 - 10, src.value + 10)")
-      }.getMessage
-      errorContains(e, "MERGE destination only supports Delta sources")
```

Comment:
Why duplicate the test if all you are changing is the check on the error message? you can modify the check by checking against two options .. message should contain either X or Y.

Reply:
fix